src/mem/mem_interface.cc

```diff
@@ -289,11 +289,13 @@ DRAMInterface::activateBank(Rank& rank_ref, Bank& bank_ref,
     // verify that we have command bandwidth to issue the activate
     // if not, shift to next burst window
     Tick act_at;
-    if (twoCycleActivate)
+    if (twoCycleActivate){
```

Comment:
Ugh... this is why you should always use curly braces! (I'm just pointing out the poor coding practice that was already there.)

You're missing a space. (Below as well)